packages/flutter/lib/src/material/data_table.dart

```diff
@@ -481,18 +556,19 @@ class DataTable extends StatelessWidget {
   final double dividerThickness;
 
   Widget _buildCheckbox({
-    Color color,
+    Color checkboxColor,
```

Comment:
Since we're renaming the parameter, it seems like calling it activeColor, would be a tiny bit more straightforward.